pkg/cmdlets/auth/login.go

```diff
@@ -240,9 +247,11 @@ func AuthenticateWithBrowser(out output.Outputer, auth *authentication.Auth, pro
 		out.Notice(locale.Tr("err_browser_open", *response.VerificationURIComplete))
 	}
 
+	var refreshToken string
```

Comment:
Please use our local terminology, I believe this is an API key? (update all references)